blocks/sidenav/sidenav.js

```diff
@@ -336,10 +349,26 @@ export default async function decorate(block) {
     block.querySelector('a[slot="document"]').textContent = book.default.data[0].title;
 
     const sorted = sortBook(book);
-    renderTOC(toc, sorted);
+    renderTOC(toc, sorted, true);
 
     addEventListeners(wrapper);
-  });
 
-  // TODO display TOC links and filter
+    // Lazy load additional books non blocking
+    Promise.all(store.additionalBooks.map((additionalBook) => new Promise((resolve) => {
```

Comment:
I'm a little hesitant to load all the books here, looks like it ends up fetching all the JSON before the header, fonts, icons and some images.. maybe we could do something like:

- load all on `load:delayed` event like how we [load mobile breadcrumbs](https://github.com/hlxsites/prisma-cloud-docs-website/blob/main/blocks/header/header.js#L537)
- load one/all on `mouseenter` of book title in sidenav
- load all on filter input `mouseenter` or `input`

wdyt?

Reply:
Yup I'll try the `load:delayed` event 👍